tests/test_cli.py

```
@@ -1291,5 +1326,1849 @@
+        # Import and call function
+        from mmrelay.cli import get_version
+        version = get_version()
+
+        # Verify results
+        self.assertIsInstance(version, str)
+        self.assertTrue(len(version) > 0)
+        # Should be in semantic version format (at least one dot)
+        self.assertIn(".", version)
+
+
+class TestGenerateSampleConfig(unittest.TestCase):
```

Ruff (0.12.2)

Redefinition of unused `TestGenerateSampleConfig` from line 321

(F811)